.github/workflows/windows-nightly-ci.yml

```diff
@@ -25,8 +25,24 @@ jobs:
         shell: powershell
         run: |
           $InstallerPath = "C:\vscode-installer.exe"
```

Comment:
@midays The VM should have vscode installed on it . @msajidmansoori12 Please correct me if I am wrong .
We don't have to install vscode on it .